argostranslate/package.py

```diff
@@ -151,7 +159,7 @@ def download(self):
         """Downloads the AvailablePackage and returns its path"""
         url = self.links[0]
         filename = self.from_code + '_' + self.to_code + '.argosmodel'
-        filepath = settings.downloads_dir / filename
```

Comment:
I find the overridden "/" operator for pathlib.Path objects is more clear than using `join`.